src/common/urlUtils.ts

```diff
@@ -157,6 +157,16 @@ export function isValidUrl(url: string): boolean {
   }
 }
 
+export function properlyResolveFileUrl(url: string): string {
```

Comment:
This isn't called anywhere, you can delete it

Reply:
Done!